lighthouse-core/lib/emulation.js

```diff
@@ -130,22 +147,60 @@ function goOffline(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', OFFLINE_METRICS);
 }
 
-function enableCPUThrottling(driver) {
-  return driver.sendCommand('Emulation.setCPUThrottlingRate', CPU_THROTTLE_METRICS);
+/**
+ * @param {Driver} driver
+ * @param {LH.ThrottlingSettings|undefined} throttlingSettings
```

Comment:
@brendankenny for some reason type checking was refusing to allow `LH.ThrottlingSettings=` as an optional parameter, no clue why any ideas?

Reply:
looks like this is a temporary bug in the version of TS we're using. I was actually pleased that it started completely differentiating between optional and `|undefined`, but looks like that wasn't on purpose :) (there's a separate discussion around using `void` or some other kind of `missing` type to indicate the "optional but never `undefined`" case: https://github.com/Microsoft/TypeScript/issues/13195).

Later releases have that fixed, though. I'm about to make a PR bumping us to `typescript@2.9.0-dev.20180323` (for reasons I'll explain there). You could get a jump on that.